riak_test/opbcounter_test.erl

```diff
@@ -33,7 +37,7 @@ confirm() ->
     decrement_test(Nodes),
     transfer_test(Nodes),
     conditional_write_test(Nodes),
-    rt:clean_cluster(Nodes),
+    %rt:clean_cluster(Nodes),
```

Comment:
I was just pointing out that being careful when designing the test, we could avoid using the call most of the times. Even saving 30 seconds in total is a great saving knowing that we run the whole test suit pretty often.

> On 14 Sep 2015, at 17:50, Christopher S. Meiklejohn <notifications@github.com> wrote:
> 
> In riak_test/opbcounter_test.erl <https://github.com/SyncFree/antidote/pull/143#discussion_r39417192>:
> 
> > @@ -33,7 +37,7 @@ confirm() ->
> >      decrement_test(Nodes),
> >      transfer_test(Nodes),
> >      conditional_write_test(Nodes),
> > -    rt:clean_cluster(Nodes),
> > +    %rt:clean_cluster(Nodes),
> but it will slow down everything
> 
> Of course it does, it's responsible for ensuring that no data exists between tests in the same module that would cause the test to fail because of some stale state. It's doing work, so of course it's going to slow things down.
> 
> Riak Test only ensures that state is clean between test modules, not between multiple tests in the same module. If you have multiple tests in the same module, of which Antidote has many, this call is required to ensure determinism.
> 
> You have two choices here: if the test is going to test multiple things and reconfigures the cluster between tests, this call is required to ensure that the state in the data directory has been reset -- otherwise you run the risk of executing something with the previous ring or reading data from a previous execution if you happen to access the same key. Your other choice is to break things out into separate test modules.
> 
> But, I don't believe the call should just be removed without proper investigation.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/143/files#r39417192>.
>